tools/swift-plugin-server/CMakeLists.txt

```diff
@@ -1,4 +1,16 @@
 if (SWIFT_BUILD_SWIFT_SYNTAX)
+  # hack: override macOS deployment target to 10.15.4
+  # since that's the minimum required by WasmKit
+  set(SWIFT_DARWIN_DEPLOYMENT_VERSION_OSX "10.15.4")
+  include(DarwinSDKs)
+  swift_get_host_triple(SWIFT_HOST_TRIPLE)
+
+  # override the remote SwiftSystem dependency in WasmKit
+  FetchContent_Declare(SwiftSystem SOURCE_DIR "${PROJECT_SOURCE_DIR}/../swift-system")
+
+  FetchContent_Declare(WasmKit SOURCE_DIR "${PROJECT_SOURCE_DIR}/../wasmkit")
+  FetchContent_MakeAvailable(WasmKit)
```

Comment:
Should we be declaring `SWIFT_PATH_TO_SWIFT_SYSTEM_SOURCE` and `SWIFT_PATH_TO_WASMKIT_SOURCE` instead of hardcoding these paths? Curious why the `SWIFT_PATH_TO_*` variables are used despite the assumption that `swift` has those dirs as siblings courtesy of `update-checkout`.